integration/container/mounts_linux_test.go

```diff
@@ -285,15 +285,15 @@ func TestContainerVolumesMountedAsShared(t *testing.T) {
 
 	// Convert this directory into a shared mount point so that we do
 	// not rely on propagation properties of parent mount.
```

Comment:
Looks like this comment was no longer matching what was actually done?
```suggestion
	// Convert this directory into a private mount point so that we do
	// not rely on propagation properties of parent mount.
```

Reply:
Actually; wondering if this last commit should be in this PR, as it doesn't look related to the change itself 😅  (not a blocker though)

Reply:
No, it's still shared. This change is just a refactoring:

> `mount.Mount(X,X, "none", "bind,private")` => `mount.MakePrivate(X)`

and

> `mount.Mount("none", X, "none", "shared")` => `mount.MakeShared(X);`

In both cases, it generates 3 mount syscalls: `bind`, `private`, `shared` on that path.

The _final_ result is a 'shared' mount (so others mounting this will see mounts done here or there), but which is not receiving any mount changes from its parent tree.

You're right, it's not part of this change. It's a minor cleanup that was pointed out to me in #41638 (where I moved these tests from integration-cli to integration) _after_ that was merged. Or rather, the comment was made before it was merged, but I didn't see the comment until after it was merged.

Reply:
Ah, yes, that's definitely confusing. I think what makes it confusing is that the comment is associated with he first of those mounts (`mount.MakePrivate`), which is only the first step; perhaps we should rephrase it somewhat to outline the steps (bind -> private -> shared).

Anyway; it's indeed not something new, so let's keep that for a follow-up